scout/models/case/case_loading_models.py

```diff
@@ -273,13 +273,15 @@ class Image(BaseModel):
 
     @field_validator("path", mode="before")
     @classmethod
-    def check_image_format(cls, path: str) -> Optional[str]:
+    def check_image_path(cls, path: str) -> Optional[str]:
         """Make sure that the image is has standard format."""
         image_format: str = path.split(".")[-1]
         if image_format not in SUPPORTED_IMAGE_FORMATS:
             raise TypeError(
                 f"Custom images should be of type: {', '.join(SUPPORTED_IMAGE_FORMATS)}"
             )
+        if "{REPID}" not in path and _is_string_path(path) is False:
```

Comment:
REPID-like images get validated when they get their real path, and the image validation is invoked at [this line](https://github.com/Clinical-Genomics/scout/blob/9e6345ddfe1816926c7e626ffc12319964a9c693/scout/models/case/case_loading_models.py#L320).

So all custom images get path-validated wither before or after their path is fixed